lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -132,6 +149,24 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         })
     }
 
+    fn try_load(
+        path: &Path,
+    ) -> OperationResult<(SparseIndexConfig, TInvertedIndex, IndicesTracker)> {
+        let stored_version = SparseVectorIndexVersion::load(path)?;
```

Comment:
WDYT if we add a "default_version" here, to prevent migration before we actually enable the new type of posting lists?